tests/appsec/iast/utils.py

```diff
@@ -24,10 +24,24 @@ def _get_span_meta(request):
     return meta
 
 
+def _get_span_meta_struct(request):
+    spans = [span for _, span in interfaces.library.get_root_spans(request=request)]
```

Comment:
Instead of fishing for the root span in `_get_span_meta_struct` and then in `_get_span_meta`, can we just get the root span, and then extract meta and meta struct? Just to avoid redundant operations.